tests/unit/test_plugins_cis_audit.py

```diff
@@ -69,37 +74,39 @@ def tearDownClass(cls):
         if os.path.exists(cls.cloud_init_logfile):
             os.remove(cls.cloud_init_logfile)
 
-    @mock.patch("files.plugins.cron_cis_audit._get_profile_by_ubuntu_series")
+    @mock.patch("files.plugins.cron_cis_audit._valid_profiles_for_platform")
     def test_get_cis_hardening_profile_default_bionic(self, mock_profiles):
         """Test hardening profile passing defaults on bionic."""
         # default profile should be return if profile passed is invalid
         mock_profiles.return_value = self.bionic_profiles
         profile = cron_cis_audit._get_cis_hardening_profile("")
+        expected_profile = "level1_server"
         self.assertEqual(
             profile,
-            self.bionic_profiles[0],
+            expected_profile,
             "Default profile should have been returned",
         )
 
-    @mock.patch("files.plugins.cron_cis_audit._get_profile_by_ubuntu_series")
+    @mock.patch("files.plugins.cron_cis_audit._valid_profiles_for_platform")
```

Comment:
It would be better to patch `_get_major_version`, so we can test `_valid_profiles_for_platform` too.